components/SummaryContainer.tsx

```diff
@@ -32,15 +32,19 @@ export default function SummaryContainer(props: {
     <div className={`${styles.summaryStyle} ${sharedStyles.sectionContainer}`}>
         <h2 className="h2NoMargins">Evergreen Dashboard</h2>
         <p className={headerStyles.headerStyle}>
-          Monitoring for <b>{org.targetOrganisation}</b> Github Organisation
+          Monitoring for <b>{config.targetOrganisation}</b> Github Organisation
         </p>
         <div>
             {props.loadingBackdrop}
         </div>
       <div className={styles.container}>
-      <div className={`${overallNan == false ? styles.summaryOverall : styles.summaryOverallGrey} ${styles.sharedCompProps} ${styles.summaryOverall}`}>
-            <h3 className={styles.overallTitleStyle}>Overall</h3>
-            <h2 className={styles.percentStyle} >{overallPercentStr}</h2>
+		<div className={`${styles.summaryComponent} ${styles.sharedCompProps}`}>
+			<h3 className={styles.summaryStyle}>Target ({config.targetPercentage}%)</h3>
+			<div className={`${overallStyle} ${overallColour} ${styles.smallSharedCompProps} ${styles.summaryOverall}`}>
```

Comment:
Looks like the headings and sub-content for the summary container page aren't aligned horizontally
<img width="632" alt="image" src="https://user-images.githubusercontent.com/81656054/188253379-22a09fc0-bb15-46ac-a421-d9fc6c0674a0.png">

Reply:
Yeah, I've just made it resemble the figma. There's supposed to be an image that pushed it to the right, but I couldn't find the mui icon.

Reply:
I've made it smaller now, and it lines up with the "Dependent Repos" box (but not the "Repos Overview" box, which is lower).